tools/ci_build/github/azure-pipelines/templates/rocm.yml

```diff
@@ -0,0 +1,154 @@
+parameters:
+- name: AdditionalBuildFlags
+  type: string
+  default: ''
+
+- name: PythonVersion  
+  type: string
+
+- name: RocmVersion  
+  type: string
+
+
+  
+jobs:
+- job: wheels_python_${{ replace(parameters.PythonVersion,'.','_') }}_rocm_${{ replace(parameters.RocmVersion,'.','_') }}
+  workspace:
+    clean: all
+  timeoutInMinutes: 180
+  pool: Ubuntu-2004-rocm-aiinfra
+  variables:
+    PythonVersion: ${{ parameters.PythonVersion }}
+
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+
+  - template: set-python-manylinux-variables-step.yml
+  - template: get-docker-image-steps.yml
+    parameters:
+      Dockerfile: tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_rocm${{ parameters.RocmVersion }}
+      Context: tools/ci_build/github/linux/docker
+      DockerBuildArgs: >-
+        --build-arg TORCH_VERSION=1.11.0
```

Comment:
PyTorch latest version is 1.12

Reply:
The rocm team decides which pytorch version to use. We will not change it.

Reply:
This file is for ROCM EP. Each EP's owners determine which CUDA version/ROCM version/... they want to use. Usually we don't make changes on it.

Reply:
They do not always want to use the latest.